mlc_llm/utils.py

```diff
@@ -53,6 +53,10 @@ def argparse_postproc_common(args: argparse.Namespace) -> None:
 
     model_conv_templates = {
         "llama-2": "llama-2",
+        "codellama-7b-instruct": "codellama_instruct",
```

Comment:
In the future, it would be nice to match the templates more smoothly, for example through regex.